templates/webapi/branding/openqa.suse.de/commenting_tools.html.ep

```diff
@@ -0,0 +1,5 @@
+% if (!$group_comment && !$job->is_ok) {
+    <a class="help_popover fa fa-unlink" title="Add marker so the specified maintenance incident will be approved despite this job"
```

Comment:
The comment is not really correct because if other tests still fail the maintenance incident won't be approved but fine for now

Reply:
I thought the "despite this job" would be sufficient.